app/models/project.rb

```diff
@@ -1,4 +1,10 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def contestant_count
+    contestants.count
```

Comment:
Great encapsulation here as well. I know it can feel silly to make a method for one minor calculation method but I would make the same choice in order to keep all data manipulation close to the db 👍